docs/tutorials.md

```diff
@@ -8,24 +8,30 @@ paths:
     - tutorials
 ---
 
-# Tutorials
+If you're new to Apify, [check out the Quick Start tutorial]({{@link tutorials/quick_start.md}}). It will show you how to [run existing actors]({{@link tutorials/quick_start.md#run-an-actor}}) and [create your first Apify actor]({{@link tutorials/quick_start.md#create-an-actor}}).
 
-In this section, you will find everything you need to begin using our platform.
+Next, learn how to [set up your first web scraping job]({{@link tutorials/apify_scrapers/getting_started.md}}) with a step-by-step guide. Then, get familiar with our main tools by following
+the [tutorial]({{@link tutorials/apify_scrapers/web_scraper.md}}) for **Web Scraper** ([apify/web-scraper](https://apify.com/apify/web-scraper)),
+the [tutorial]({{@link tutorials/apify_scrapers/cheerio_scraper.md}}) for **Cheerio Scraper** ([apify/cheerio-scraper](https://apify.com/apify/cheerio-scraper)),
+or the [tutorial]({{@link tutorials/apify_scrapers/puppeteer_scraper.md}}) for **Puppeteer Scraper** ([apify/puppeteer-scraper](https://apify.com/apify/puppeteer-scraper))
 
-If you're new to Apify, you can familiarize yourself with the basics of our ready-made web scraping tools [here]({{@link tutorials/apify_scrapers.md}}).
+## Building actors
 
-Our step-by-step guides will walk you through setting up your [first web scraping job]({{@link tutorials/apify_scrapers/getting_started.md}}),
-after which you can get familiar with our tools by reading
-the [tutorial]({{@link tutorials/apify_scrapers/web_scraper.md}}) for our **Web Scraper** ([apify/web-scraper](https://apify.com/apify/web-scraper)),
-the [tutorial]({{@link tutorials/apify_scrapers/cheerio_scraper.md}}) for our **Cheerio Scraper** ([apify/cheerio-scraper](https://apify.com/apify/cheerio-scraper)),
-or the [tutorial]({{@link tutorials/apify_scrapers/puppeteer_scraper.md}}) for our **Puppeteer Scraper** ([apify/puppeteer-scraper](https://apify.com/apify/puppeteer-scraper)).
+Building your own actors requires some experience with coding, however the language doesn't matter. Apify supports all programming languages. Below are guides for the most popular ones.
```

Comment:
Since we're saying that we support all languages, we could maybe have some docs page about actors really being Docker images, to show that anything that can run in a Docker image can run on Apify?
Or maybe at least link this page: https://docs.apify.com/actors/development

Reply:
I dunno if linking to the development page will help here - it's not written there explicitly (or anywhere actually 🤷) 

Do you think it's not enough to just say can support any language?